website/docs/r/api_management.html.markdown

```diff
@@ -185,12 +184,14 @@ A `security` block supports the following:
 
 ---
 
-A `sku` block supports the following:
+A `sku` block supports the following: (Deprecated)
 
-* `name` - (Required) Specifies the Pricing Tier for the API Management Service. Possible values include: `Developer`, `Basic`, `Standard` and `Premium`.
+* `name` - (Required) Specifies the Pricing Tier for the API Management Service. Possible values include: Developer, Basic, Standard and Premium.
 
 * `capacity` - (Required) Specifies the Pricing Capacity for the API Management Service.
 
+-> **Note:** This property has been deprecated in favour of the `sku_name` property and will be removed in version 2.0 of the provider.
```

Comment:
Could we add this to the upgrading to 2.0 guide found in the guides directory?

Reply:
Sure... added both the data source and the resource.